doc/guides/dts/index.rst

```diff
@@ -196,61 +269,58 @@ setting the bus speed:
    };
 
 The ``&can0 { ... };`` syntax adds/overrides properties on the node with label
-``can0``, i.e. the ``can@4002400`` node.
+``can0``, i.e. the ``can@4002400`` node defined in the :file:`.dtsi` file.
 
 Other examples of board-specific customization is pointing properties in
 ``aliases`` and ``chosen`` to the right nodes (see :ref:`dt-alias-chosen`), and
-making GPIO/PinMux assignments.
+making GPIO/pinmux assignments.
 
 Currently supported boards
 **************************
 
-Device tree is currently supported on all embedded targets except posix
+Devicetree is currently supported on all embedded targets except posix
 (boards/posix) and qemu_x86_64 (boards/x86_64).
 
 Adding support for a board
 **************************
 
-Adding device tree support for a given board requires adding a number of files.
+Adding devicetree support for a given board requires adding a number of files.
 These files will contain the DTS information that describes a platform, the
-YAML descriptions that define the contents of a given Device Tree peripheral
-node, and also any fixup files required to support the platform.
+bindings in YAML format, and any fixup files required to support the platform.
 
-When writing Device Tree Source files, it is good to separate out common
-peripheral information that could be used across multiple SoC families or
-boards.  DTS files are identified by their file suffix.  A .dtsi suffix denotes
-a DTS file that is used as an include in another DTS file.  A .dts suffix
-denotes the primary DTS file for a given board.
+It is best practice to separate common peripheral information that could be
+used across multiple cores, SoC families, or boards in :file:`.dtsi` files,
+reserving the :file:`.dts` suffix for the primary DTS file for a given board.
 
-The primary DTS file will contain at a minimum a version line, optional
-includes, and the root node definition.  The root node will contain a model and
-compatible that denotes the unique board described by the .dts file.
+Devicetree Source File Template
+===============================
 
-Device Tree Source File Template
-================================
+A board's :file:`.dts` file contains at least a version line, optional
+includes, and a root node definition with ``model`` and ``compatible``
+properties. These property values denote the particular board.
 
 .. code-block:: none
 
-  /dts-v1/;
+   /dts-v1/;
 
-  / {
-          model = "Model name for your board";
-          compatible = "compatible for your board";
-          /* rest of file */
-  };
+   #include <vendor/soc.dtsi>
 
+   / {
+           model = "Human readable board name";
+           compatible = "vendor,soc-on-your-board's-mcu";
+           /* rest of file */
```

Comment:
(DTS supports the nicer `// rest of file` format too.)

Reply:
C89 style is what virtually all (all but three) of the .dts and .dtsi files I found in the tree are using, and Zephyr's C comment style is `/* */`, so I'm going to leave this as-is unless you're proposing a tree-wide change to C99 / C++ style single line comments.